functions/templates/bulk-shopping-volunteer-email.md.mustache

```diff
@@ -0,0 +1,54 @@
+Hi {{firstName}},
+
+Thank you again for volunteering to shop for these custom items!
+
+Please sort the items so that each ticket ID has its own bag or bags, and label
+each one of the bags with its corresponding **route number and ticket ID**.
+
+There are a variety of ways you can label the bags:
+
+1. Write the route number and ticket ID on a slip of paper or post-it note and
+   staple or tape it to the bag.
+2. Write the route number and ticket ID on a slip of paper or post-it and just
+   set it inside the bag with the groceries.
+3. Write the route number and ticket ID directly on the grocery bag(s) with a
+   Sharpie or other pen/marker that won't smear or rub off.
+
+Please remember to bring at least one pen and some paper with you to the grocery
+store! We need drivers to drop these items off at the warehouse, **[221 Glenmore
+Ave, Gate 4]({{{warehouseMapsUrl}}}) by 12pm**, so please plan your shopping and a
+meetup time with your car teammate accordingly.
+
+Submit the [reimbursement form](https://airtable.com/shrvHf4k5lRo0I8F4) in the
+usual way, with the total amount you spent and images of your receipt(s). Just
+pick one ticket ID from your list and enter that. Please add a note that your
```

Comment:
"pick one ticket ID from your list and enter that": is there any way we can improve this flow?

Reply:
Filed #86